package.json

```diff
@@ -40,6 +41,7 @@
     "redux": "^3.4.0",
     "redux-actions": "^0.9.1",
     "redux-thunk": "^2.0.1",
+    "svg-inline-react": "^3.1.0",
```

Comment:
I think there are better ways to load SVG. It looks like this project and `svg-inline-loader` were created as a work-around in the past when React didn't properly support SVG, and it is just being maintained now. (https://github.com/sairion/svg-inline-react/issues/15#issuecomment-264492050).

A quick search looks like importing SVG into React has a messy history, but I think it would be safe to do it like they do in `create-react-app`, e.g. with [svgr](https://github.com/smooth-code/svgr).

Reply:
Ah great, I was wondering about that! SVGR looks good - I'll try that out, thanks!